### PR TITLE
Refactor: code

### PR DESCRIPTION
**Pull request by Nasfame**

### CODE REVIEW

File: .run/streamlit run.run.xml
Comment (Graceliying82) [inline .run/streamlit run.run.xml:1]: This configuration file for is your IDE, right? I don't feel it's necessary in the repo. Can you put the .run into .gitignore?

File: .env.example
Comment (Graceliying82) [inline .env.example:1]: Good idea. I planned to add it to the readme first.

File: app.py
Comment (Graceliying82) [inline app.py:1]: Thank you. These changes look good to me.

**Review by Graceliying82 — CHANGES_REQUESTED** [review]
I would suggest remove .run or put it into .gitignore. Other than that, the code change looks great to me. Thank you.

File: .run/streamlit run.run.xml
Comment (Nasfame) [reply .run/streamlit run.run.xml:1]: @Graceliying82 you were using Pycharm as well. So i left it there. Never mind i ill replace it with a Makefile

**Review by Nasfame — COMMENTED** [review]
LGTM

**Comment by Nasfame** [comment]
For https://github.com/Graceliying82/pdf-chat/issues/2

**Comment by Nasfame** [comment]
For #3

**Comment by Graceliying82** [comment]
Nice job, Hiro.

**Pull request merged**